src/App.css

```diff
@@ -6,6 +6,17 @@
   padding: 0;
   font-size: 30px;
   font-family: 'PT Sans Narrow', sans-serif;
+  color: var(--black);
+
+  /* color palette for quick access named by approximate colors */
+  --blue: #3ec1d3;
+  --white: #f6f7d7;
+  --yellow: #ff9a00;
+  --pink: #ff165d;
+  --brown: #d38d3e;
+  --red: #d3433e;
+  --black: #1a0807;
+  --green: #87d697;
```

Comment:
This. Is. Great. 👏

Reply:
Thanks, I love using variables to make the color palette easy to change!